lib/src/database/db.dart

```diff
@@ -209,7 +210,7 @@ class Db {
       }
 
       if (connection == null) {
-        connection = _masterConnection;
+        connection = _masterConnectionVerified;
```

Comment:
Yeahhh.. thinking about it, you're doing as assignment, I don't like queries that have side effects. Leave it like this.